web/src/components/map/index.js

```diff
@@ -46,10 +48,33 @@ export const Map = () => {
   const [pollutants, setPollutants] = useState(EMPTY_POLLUTANTS);
   const [isLoadingPollutants, setIsLoadingPollutants] = useState(false);
 
+  /**
+   * Load a Set of all dates with at least one collection
+   * @modifies {api} calls to get one collection per date that has any
+   * @modifies {setOfDates}
+   * @modifies {mapDate}
+   * @returns {axios.CancelToken.source().cancel} cancel the api call
+   */
+  useEffect(() => {
+    const source = axios.CancelToken.source();
+    (async () => {
+      const { listOfDates, thrownCode } = await getAllDates(source);
```

Comment:
The cool kids indicate lists by making the variable name plural. (I understand that you're de-structuring an object and the first place to change it is actually the return object for the function).

```suggestion
      const { allDates, thrownCode } = await getAllDates(source);
```

Reply:
Thanks! I'll change it in index.js and in the called function in utils.js.